network/endpoint_windows.go

```diff
@@ -321,6 +321,16 @@ func (nw *network) configureHcnEndpoint(epInfo *EndpointInfo) (*hcn.HostComputeE
 		return nil, err
 	}
 
+	// add hcnEndpoint policy for accelnet
+	if epInfo.NICType == cns.NodeNetworkInterfaceAccelnetFrontendNIC {
+		endpointPolicy, err := policy.AddAccelnetPolicySetting()
```

Comment:
add UT when this fails?

Reply:
added a test case to call AddAccelnetPolicySetting() in policy_windows_test.go; this func is to generate a fixed policy and we can only add an UT to make sure this func works...